app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,26 @@
+class TasksController < AuthenticatedController
+  include ProjectScoped
+  include TasksHelper
+
+  skip_before_action :set_project, if: -> { params[:project_id].blank? }
+  skip_before_action :set_nodes, if: -> { params[:project_id].blank? }
+
+  def index
+    @default_columns = ['Title', 'Methodology', 'Due Date', 'Assigned']
+    @tasks = assigned_cards
+
+    if params[:project_id].present?
+      @local_storage_key = "project.ce.project_#{current_project.id}.tasks_datatable"
+    end
+  end
+
+  protected
```

Comment:
Any reason for choosing `protected` over `private` here?

Reply:
flipped to `private` I'm not sure why I went with `protected` it's possible I got it from the notifications controller (was exploring that implementation initially)